script/Arcueid/Day 4/04_00_ARC04_6_2.txt

```diff
@@ -306,7 +306,7 @@ I never thought I would see something like this in my lifetime.
 [sha:6b0c313d7f3ac0d7fa52f04b9c1603c56c49dc01]{
 -- Page 443, Offset 6786.
 -- 「…………はあ」
-"......Hah."
+"......%{i}Sigh%{/i}."
```

Comment:
check in game but i don't think this one is actually a sigh. More like an exasperated exhale. Might need to go back to Hah but you can put it in italics.

Reply:
i went back to it but i'm not sure what's the difference

Reply:
I'd say safer to stick to Haa fwiw